sawtooth-sdk-transaction-processor/src/test/java/sawtooth/sdk/processor/TransactionHandlerTaskTest.java

```diff
@@ -0,0 +1,84 @@
+package sawtooth.sdk.processor;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import org.junit.Test;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import sawtooth.sdk.messaging.Stream;
+import sawtooth.sdk.processor.exceptions.InternalError;
+import sawtooth.sdk.processor.exceptions.InvalidTransactionException;
+import sawtooth.sdk.protobuf.Message;
+import sawtooth.sdk.protobuf.Message.MessageType;
+import sawtooth.sdk.protobuf.TpProcessResponse;
+
+public class TransactionHandlerTaskTest {
+
+  TpProcessResponse response;
+
+  @Test
+  public void testRun() {
```

Comment:
Comments here that describe the test would be helpful. In other places, we have found it helpful to add a comment with a short description of the test (i.e. what it's testing) followed by the abstract steps the test is performing.

The recent PBFT tests are a good example.

Reply:
fixed here, the other test needs adjusting for determinacy, so I'll do that first.

Reply:
Looks like this did not make it into the current set of commits.